config/manifests/bases/cass-operator.clusterserviceversion.yaml

```diff
@@ -16,33 +22,145 @@ spec:
       kind: CassandraDatacenter
       name: cassandradatacenters.cassandra.datastax.com
       version: v1beta1
-  description: '-'
-  displayName: DataStax Apache Cassandra Operator
+  description: ../../docs/developer/csv-description.md
+  displayName: DataStax Kubernetes Operator for Apache Cassandra
   icon:
-  - base64data: ""
-    mediatype: ""
+  - base64data: |
```

Comment:
Is there anyway we could store this in a separate file and include it similar to the `spec.description` field?

Bonus points if we don't have to store it in base64 format and can just read in the binary PNG.

Reply:
That should be possible with postprocessing (and setting with yq), but I don't have the png file.